Feature/index page initialization

Jaidenpearson opened the pull request:
- Loaded bootstrap theme/template and implemented files in their respective directories. 
- Server configured to pull static files from /public
- Deleted scss files from skeleton repo

Commit:
deleted SASS references/files, implemented bootstrap to index

Commit:
bootstrap theme implemented properly this time. styles located in /styles and compile to /public. Server configured to pull static files from /public

Commit:
bootstrap theme implemented properly this time. styles located in /styles and compile to /public. Server configured to pull static files from /public

Commit:
fixed conflicts

Gladdy17 reviewed (commented):
@Jaidenpearson this is looking good
i pulled down the changes. 
for some reason js does not seem to be loading is that something we will add the next pr

Commit:
script files not referenced properly. JS now loads and confirms with a console.log in the browser

Jaidenpearson commented:
JS files weren't referenced properly so I changed the src tags and moved the bootstrap from its original folder to /public/scripts where we will keep all our front end JS

Gladdy17 reviewed (approved):
Good to merge brother its looking good
